scripts/generatePortVersionsDb.py

```diff
@@ -0,0 +1,76 @@
+import os
+import os.path
+import sys
+import subprocess
+import json
+import time
+
+from subprocess import CalledProcessError
+from json.decoder import JSONDecodeError
+from pathlib import Path
+
+def make_dir(dir_path):
+    if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+
+
+def get_current_git_ref():
+    try: 
+        output = subprocess.run(args=['git.exe', 'rev-parse', '--verify', 'HEAD'], capture_output=True)
+        if output.returncode == 0:
+            return output.stdout.decode('utf-8').strip()
+        return ''
+    except CalledProcessError as err:
+        print("Failed to get git ref")
+
+
+def generate_port_versions_db(ports_path, db_path, revision):
+    start_time = time.time()
+
+    port_names = [item for item in os.listdir(ports_path) if os.path.isdir(os.path.join(ports_path, item))]
+    total_count = len(port_names)
+    for counter, port_name in enumerate(port_names):
+        containing_dir = os.path.join(db_path, port_name[0])

```

Comment:
```suggestion
        containing_dir = os.path.join(db_path, f'{port_name[0]}-')
```

don't forget the `-` :)